### PR TITLE
starting with graphql with locations

### PR DESCRIPTION
**Pull request by FredPeal**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "require-dev": {
         "fakerphp/faker": "^1.9.1",
         "kitloong/laravel-migrations-generator": "^5.2",
+        "mll-lab/laravel-graphql-playground": "^2.6",
```
Comment (rwhite27) [inline composer.json:31]: Do we really need this ? @FredPeal

File: graphql/schema.graphql [hunk 2]
```diff
@@ -2,4 +2,5 @@
 scalar DateTime @scalar(class: "Nuwave\\Lighthouse\\Schema\\Types\\Scalars\\DateTime")
 
 #import hello.graphql
+#import countries.graphql
```
Comment (kaioken) [inline graphql/schema.graphql:5]: @FredPeal move to folder schema

**Review by kaioken — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (FredPeal) [reply composer.json:31]: nop

File: app/GraphQL/Mutations/CreateCountry.php [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+use Illuminate\Support\Arr;
+use Kanvas\Locations\Countries\Models\Countries;
+
+final class CreateCountry
+{
+    /**
+     * @param  null  $_
+     * @param  array{}  $args
+     */
+    public function __invoke($_, array $args): Countries
```
Comment (kaioken) [inline app/GraphQL/Mutations/CreateCountry.php:13]: @FredPeal chance args to $request so it easier to understan

**Review by kaioken — CHANGES_REQUESTED** [review]

**Review by kaioken — APPROVED** [review]

**Pull request merged**